content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -4,5 +4,10 @@ title: What´s new
 layout: bundle
 ---
 
-
 Release 10.15.0 includes the following new features or major feature enhancements.
+
+### New All devices view
+
+In an effort to provide users a more consistent experience across the platform the **All devices** view has been re-shaped. It is now implemented in the data grid fashion users know from **Subassets** and **Search** views. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
```

Comment:
users > you
**All devices** view > **All devices** page
users know > you already are familiar with
views > pages